pkg/registry/core/pod/strategy.go

```diff
@@ -644,28 +644,20 @@ func validateContainer(container string, pod *api.Pod) (string, error) {
 
 // applySeccompVersionSkew implements the version skew behavior described in:
 // https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/135-seccomp#version-skew-strategy
+// Note that we dropped copying the field to annotation synchronization in
+// v1.25 with the functional removal of the annotations.
 func applySeccompVersionSkew(pod *api.Pod) {
 	// get possible annotation and field
 	annotation, hasAnnotation := pod.Annotations[v1.SeccompPodAnnotationKey]
-	field, hasField := (*api.SeccompProfile)(nil), false
+	_, hasField := (*api.SeccompProfile)(nil), false
```

Comment:
```suggestion
	hasField := false
```

Reply:
Alternatively:
```suggestion
	hasField := (pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.SeccompProfile != nil)
```

Reply:
Going for simplicity and applying the first suggestion. Thanks for that!